dojo/tools/checkmarx_one/parser.py

```diff
@@ -107,4 +107,31 @@ def get_findings(self, file, test):
                             vulnerability.get("similarityId")
                         )
                     findings.append(finding)
+                elif result_type in ["sca", "sca-container"]:
+                    description = vulnerability.get("description")
+                    finding = Finding(
+                        title=f"{description}",
```

Comment:
```suggestion
                        title=description,
```

Reply:
Actually, this should probably use a combination of the vulnerability ID (e.g. CVE-x-x) and the package name/identifier instead of the description.

Reply:
https://github.com/DefectDojo/django-DefectDojo/pull/9917/commits/fac29815725777062de6cf66fc989f862ff70a01